pom.xml

```diff
@@ -347,28 +368,33 @@
         <commons.bean.utils.version>1.9.3</commons.bean.utils.version>
         <json-smart.version>2.4.10</json-smart.version>
         <org.wso2.orbit.nimbus.version>7.3.0.wso2v1</org.wso2.orbit.nimbus.version>
+        <encoder.wso2.version>1.2.0.wso2v1</encoder.wso2.version>
         <oltu.version>1.0.0.wso2v3</oltu.version>
+        <jackson.databind.version>2.16.1</jackson.databind.version>
         <!-- Base product dependency versions -->
         <carbon.identity.framework.version>7.0.75</carbon.identity.framework.version>
         <carbon.identity.framework.version.range>[7.0.75, 8.0.0)</carbon.identity.framework.version.range>
         <identity.inbound.auth.oauth.version>7.0.26</identity.inbound.auth.oauth.version>
         <identity.inbound.auth.oauth.version.range>[7.0.26, 7.0.62)</identity.inbound.auth.oauth.version.range>
         <!-- Unit Test Dependency versions -->
         <jacoco.version>0.8.6</jacoco.version>
+        <org.testng.version>7.3.0</org.testng.version>
         <commons-dbcp.version>1.4</commons-dbcp.version>
         <orbit.version.h2.engine>1.2.140.wso2v3</orbit.version.h2.engine>
         <mockito.version>2.28.2</mockito.version>
         <powermock.version>2.0.9</powermock.version>
         <jacoco.version>0.8.6</jacoco.version>
-        <org.testng.version>7.3.0</org.testng.version>
         <commons-dbcp.version>1.4</commons-dbcp.version>
         <orbit.version.h2.engine>1.2.140.wso2v3</orbit.version.h2.engine>
         <!-- OSGi Version ranges -->
         <osgi.framework.imp.pkg.version.range>[1.7.0, 2.0.0)</osgi.framework.imp.pkg.version.range>
         <osgi.service.component.imp.pkg.version.range>[1.2.0, 2.0.0)</osgi.service.component.imp.pkg.version.range>
+        <identity.inbound.auth.oauth.version.range>[7.0.26, 7.0.62)</identity.inbound.auth.oauth.version.range>
```

Comment:
i guess we can omit this and below line as they are already defined.

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/4/commits/98992be82ffa4483d8c6f51e4dce81ee11bb9d3b